addons/crm/tests/test_crm_lead_notification.py

```diff
@@ -12,7 +12,9 @@ class NewLeadNotification(TestCrmCommon):
     @users('user_sales_manager')
     def test_lead_message_get_suggested_recipient(self):
         """ Test '_message_get_suggested_recipients' and its override in lead. """
-        lead_format, lead_multi, lead_from, lead_partner = self.env['crm.lead'].create([
+        partner_no_email = self.contact_2
```

Comment:
Why not creating a partner in the test, just to control data ? We can define new data in a test when it is more easier to understand (and we avoid side effect of "global" test data).